src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/BullRunnerConfigExtract.java

```diff
@@ -1,33 +1,48 @@
 package edu.usf.cutr.gtfs_realtime.bullrunner;
 
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
 import java.io.BufferedReader;
 import java.io.FileReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.HashMap;
 
-import org.json.JSONArray;
-import org.json.JSONException;
-import org.json.JSONObject;
-import org.json.JSONTokener;
-
 
 public class BullRunnerConfigExtract {
     //private URL _url;
-    private static final String path2tripsFile = "../bullrunner-gtfs/trips.txt";
-    private static final String path2calFile = "../bullrunner-gtfs/calendar.txt";
-    private static final String path2routeFile = "../bullrunner-gtfs/routes.txt";
-    private static final String path2stopTimesFile = "../bullrunner-gtfs/stop_times.txt";
-    private static final String path2frequenciesFile = "../bullrunner-gtfs/frequencies.txt";
-    /**
-     * @param url
-     * the URL for the SEPTA vehicle data API.
-     */
+    private static String path2tripsFile ;
+    private static String path2calFile;
+    private static String path2routeFile;
+    private static String path2stopTimesFile ;
+    private static String path2frequenciesFile;
+
 //	public void setUrl(URL url) {
 //		_url = url;
 //	}
 
+    public void findPaths() {

```

Comment:
Could you add Javadocs to this method just to explain what it's doing (searching for files in current path first, then in parent folder)?